components/shop/shop.html

```diff
@@ -0,0 +1,106 @@
+<div class="container shop-container">
+  <!-- Header Section -->
+  <div
+    class="row shop-container--header align-items-center justify-content-between"
+  >
+    <div class="col-3 shop-container--header--sidebar d-flex flex-column">
+      <h2 class="shop-container--header--sidebar--title">Laptops</h2>
+      <p class="shop-container--header--sidebar--info">
+        About 9,620 results (0.62 seconds)
+      </p>
+    </div>
+    <div
+      class="col-xl-8 col-lg-7 col-8 shop-container--header--products-container d-flex justify-content-center align-items-start"
+    >
+      <div class="d-flex shop-container--header--products-container--sort">
+        <span class="shop-container--header--products-container--sort--title"
```

Comment:
![4r](https://github.com/user-attachments/assets/53deac00-a313-4015-8d08-bac3c4a741ae)
the not same in design

Reply:
done.